src/Command/TestCommand.php

```diff
@@ -10,7 +10,7 @@
 
 class TestCommand extends Command
 {
-    protected static $defaultName = 'mrpixwerepack:test';
+    protected static string $defaultName = 'mrpixwerepack:test';
```

Comment:
Funktioniert das noch auf der 6.6? Ich meine Symfony hat das rausgeworfen, so dass man den Namen nur noch entweder über das `AsCommand` Attribute oder den Tag in der services.xml angeben kann

Reply:
Guter Punkt 👍🏼 
Dachte das war noch über deprecated möglich um SW 6.5 und 6.6 abzudecken.
Geht tatsächlich bei der 6.6 nicht mehr, habe es nun korrigiert :)